themes/default/content/docs/guides/continuous-delivery/github-app.md

```diff
@@ -47,10 +47,10 @@ configure it to only be used by certain repositories.
 
 ![Configuration Page](/images/docs/github-app/org-configuration.png)
 
-The Pulumi GitHub application does not have access to your source code. It will only report
-status on pushes / pull requests that happen for repositories it is configured to access. You can
-also uninstall the GitHub application at any time without impacting your stacks or
-other Pulumi-managed resources.
+The GitHub app will report Stack operations on GitHub commits and pull requests
```

Comment:
why are we droping Pulumi from before GitHub?

Reply:
Typo, re-added.